cycle_docker_compose.sh

```diff
@@ -6,6 +6,7 @@ sleep 1
 
 export TRAVIS_BUILD_NUMBER=$(git describe --always --dirty)
 export TRAVIS_BRANCH=$(git rev-parse --abbrev-ref HEAD | sed "s/[^[a-zA-Z0-9]]//")
+export TRAVIS_BRANCH_SAFE=$(echo $TRAVIS_BRANCH | sed "s/[/]/-/")
```

Comment:
I think that this:

```
export TRAVIS_BRANCH=$(git rev-parse --abbrev-ref HEAD | sed "s/[^[a-zA-Z0-9]]//")
export TRAVIS_BRANCH_SAFE=$(echo $TRAVIS_BRANCH | sed "s/[/]/-/")
```

Can be simplified to:
```
export TRAVIS_BRANCH_SAFE=$(git rev-parse --abbrev-ref HEAD | sed "s/[^[a-zA-Z0-9]]//")
```

As this is only used in development and `TRAVIS_BRANCH_SAFE` is now set in travis.yml

Reply:
I have removed the `TRAVIS_BRANCH` variable, but kept the `s/[/]/-/` replacement as I believe it is still needed, even for development.